src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1134,7 +1163,20 @@ bool entitiesQuery
 
         /* Attributes part */
 
-        std::vector<BSONElement> queryAttrV = r.getField(ENT_ATTRS).Array();
+        std::vector<BSONElement> queryAttrV;
+
+        try
```

Comment:
The early detection of $err makes this try/catch mostly unnecessary. However, I think we should keep it, as it protects from weird out-of-band manipulation of the DB (e.g. somebody modifying entites documentes removing the `attrs` field) with some modifications:

1. Add a comment explaining why the try/catch is here, as we could easily forget its purpose. Something like this: "This try/catch ward actually is not needed (as every document in the entities collections has an attrs array from creation time) if we assume that out-of-band manipulation of the DB will not happen. However, it adds an extra protection measure in the case of such manitpulations could happen, due to unknown reasons" or something similar.
2. If the problem is raised, it is a serious thing that need to be researched. Thus, I'd add a LM_E in the catch clause to mark this and probably also I'd return a 500 Internal Error instead of SccContextElementNotFound.